Update init.rst cycled coupled IC info

KateFriedman-NOAA opened the pull request:
**Description**

The cycled ATM w/ coupled (S2S) model ICs were updated and the folder was renamed from "C48mx500" to "C48C48mx500". Also add Jet location.

**Type of change**

Documentation update related to non-repo change to staged ICs on supported platforms.

KateFriedman-NOAA committed:
Update init.rst cycled coupled IC info

The cycled ATM w/ coupled (S2S) model ICs were updated
and the folder was renamed from "C48mx500" to "C48C48mx500"

Refs #1552

WalterKolczynski-NOAA reviewed (changes requested):
Should add a note that for now the members are just dummy duplicates of the deterministic.

KateFriedman-NOAA committed:
Update init.rst to add note

Added note to mention that the EnKF member ICs are
currently just dummy duplicates of the deterministic.

KateFriedman-NOAA commented:
> Should add a note that for now the members are just dummy duplicates of the deterministic.

Good call...done at [77b549d](https://github.com/NOAA-EMC/global-workflow/pull/1562/commits/77b549dbb02dabbfb6a4c64e0c928bb9a7fde04d).

aerorahul commented on docs/source/init.rst:
```diff
@@ -103,79 +103,114 @@ Start date = 2021122018
 *********************************
 
-Warm-start cycled w/ coupled (S2S) model C48 atmosphere 5 degree ocean/ice ICs are available in the following locations on supported platforms:
+Warm-start cycled w/ coupled (S2S) model C48 atmosphere C48 enkf (80 members) 5 degree ocean/ice ICs are available in the following locations on supported platforms:
```
```suggestion
Warm-start cycled w/ coupled (S2S) model at C48 atmosphere and 5 degree ocean/ice ICs for deterministic and ensemble (80 members) are available in the following locations on supported platforms:
```

aerorahul commented on docs/source/init.rst:
```diff
@@ -103,79 +103,114 @@ Start date = 2021122018
 Start date = 2021032312
 
+.. note::
+   The EnKF member ICs are dummy duplicates of the deterministic at the moment.
```
```suggestion
   The ensemble ICs are dummy duplicates of the deterministic at the moment.
```

aerorahul reviewed (approved):
Just a clarification ... replace EnKF members with ensemble members.
